source/platform/renderers/Irrlicht/Irrlicht.cpp

```diff
@@ -0,0 +1,253 @@
+#include "gwork/Renderers/Irrlicht.h"
+
+#pragma comment(lib, "Irrlicht.lib")
+
+#include <iostream>
+#include <map>
+
+namespace Gwk
+{
+    namespace Renderer
+    {
+        //
+        //  Resource Loader
+        Font::Status IrrlichtResourceLoader::LoadFont(Font& font)
+        {
+            printf("Load Font\n");
```

Comment:
Do we need these? Was this for debugging? There is a logging system now, e.g.

    Gwk::Log::Write(Log::Level::Error, "Font file not found: %s", filename.c_str());

Reply:
Yes this was for debugging, I'll go ahead and get rid of it!